atomix/cluster/src/main/java/io/atomix/raft/impl/RaftContext.java

```diff
@@ -433,21 +432,23 @@ public void removeCommitListener(final RaftCommitListener commitListener) {
    * <p>Note that it will be called on the Raft thread, and as such should not perform any heavy
    * computation.
    *
-   * @param raftCommittedEntryListener the listener to add
+   * @param raftApplicationEntryCommittedPositionListener the listener to add
    */
   public void addCommittedEntryListener(
-      final RaftCommittedEntryListener raftCommittedEntryListener) {
-    committedEntryListeners.add(raftCommittedEntryListener);
+      final RaftApplicationEntryCommittedPositionListener
```

Comment:
:thought_balloon: I almost commented earlier that the name is getting long, but now with the params also changing, I love it :smile: We can go further! :smile: